### ISSUE TITLE
[Bug]: POS View 

### ISSUE THREAD

**Issue by elvece** [btcpayserver/btcpayserver#5203]
### What is your BTCPay version?

v1.11.0+093206cf1 

### How did you deploy BTCPay Server?

Docker deployment

### What happened?

After updating to v1.11.0, our POS store has the following issues:

1. Browser console error `TypeError: this._config is undefined` (screenshot included)
2. We have a product list with cart as the selected POS style; however, the cart not displaying when an item is added like it did previously (screenshot included)
3. Some list items do not display properly (see screenshot)

### How did you encounter this bug?

1. Updated instance via UI flow ie. Server Settings > Maintenance > Update
2. Refreshed store

### What browser do you use?

Firefox 115.0.3

### Additional information

<img width="1728" alt="Screenshot 2023-07-27 at 5 23 04 PM" src="https://github.com/btcpayserver/btcpayserver/assets/12953208/4c09df8b-cd9d-47e6-9ba2-f41387e3cfb9">
<img width="1728" alt="Screenshot 2023-07-27 at 5 23 16 PM" src="https://github.com/btcpayserver/btcpayserver/assets/12953208/1514b036-dff1-40e1-b3a0-aee3ece40052">
<img width="1726" alt="Screenshot 2023-07-27 at 5 23 26 PM" src="https://github.com/btcpayserver/btcpayserver/assets/12953208/3c4e802d-302d-44ab-8a94-62f16fc8d833">
<img width="1728" alt="Screenshot 2023-07-27 at 5 23 31 PM" src="https://github.com/btcpayserver/btcpayserver/assets/12953208/4bc7a385-b545-4b3f-ac3b-de4d7bfac5d5">

### Are you sure this is a bug report?

- [X] I confirm this is a bug report

**Comment by dstrukt** [comment]
cc @dennisreimann 

Thanks for the report! 

Will drop this in the MM as well.

**Comment by elvece** [comment]
Sure thing! Ok sounds good, lmk if you need any additional info.

**Comment by elvece** [comment]
What is the cleanest way to downgrade versions so we can get our store working again?

**Comment by joycevolner3** [comment]
I guess have clean out sale

**Comment by Kukks** [comment]
PLease provide me the items json code so I can replicate
![](https://i.imgur.com/fKd7F6F.png)

**Comment by elvece** [comment]
> PLease provide me the items json code so I can replicate ![](https://camo.githubusercontent.com/ae8ec16f06327ab7c3d9faebbfa37686d79bdbce4a48783ee8c039c324b2aab6/68747470733a2f2f692e696d6775722e636f6d2f664b64374636462e706e67)

<details><summary>Items JSON</summary>

```
[
  {
    "description": "Intel Core i7 (4 Cores, 8 Threads, up to 4.9GHz) CPU with 32GB (2 x 16GB) DDR4 RAM, and 2TB M.2 NVMe storage. Runs StartOS (Pure version) with Intel Management Engine disabled!",
    "id": "server-pro-2tb",
    "image": "https://btcpay.start9.com/Storage/2f8cefbd-66a5-4f24-89bd-2ebdb1e7af87",
    "priceType": "Fixed",
    "price": "1499",
    "title": "Server Pure (2TB)",
    "inventory": 2,
    "disabled": false,
    "index": "0"
  },
  {
    "description": "Intel Core i7 (4 Cores, 8 Threads, up to 4.9GHz) processor with 32GB (2 x 16GB) DDR4 RAM and 4TB M.2 NVMe storage. Runs StartOS (Pure version) with Intel Management Engine disabled!",
    "id": "server-pro-4tb",
    "image": "https://btcpay.start9.com/Storage/2f8cefbd-66a5-4f24-89bd-2ebdb1e7af87",
    "priceType": "Fixed",
    "price": "1589",
    "title": "Server Pure (4TB)",
    "inventory": 1,
    "disabled": false,
    "index": "1"
  },
  {
    "description": "11th generation Intel Celeron N4505 (2 core, 2 thread, up to 2.90GHz) processor with 16GB (2 x 8GB) DDR4 RAM and 1TB Crucial M.2 NVMe storage.",
    "id": "server-one-1tb",
    "image": "https://btcpay.start9.com/Storage/8a9cd770-d1eb-40fc-b263-5da210bdb2bc",
    "priceType": "Fixed",
    "price": "499",
    "title": "Server One (1TB)",
    "disabled": false,
    "index": "2"
  },
  {
    "description": "11th generation Intel Celeron N4505 (2 core, 2 thread, up to 2.90GHz) processor with 16GB (2 x 8GB) DDR4 RAM and 2TB Crucial M.2 NVMe storage.",
    "id": "server-one-2tb",
    "image": "https://btcpay.start9.com/Storage/8a9cd770-d1eb-40fc-b263-5da210bdb2bc",
    "priceType": "Fixed",
    "price": "563",
    "title": "Server One (2TB)",
    "disabled": false,
    "index": "3"
  },
  {
    "description": "11th generation Intel Celeron N4505 (2 core, 2 thread, up to 2.90GHz) processor with 16GB (2 x 8GB) DDR4 RAM and 4TB Crucial M.2 NVMe storage.",
    "id": "server-one-4tb",
    "image": "https://btcpay.start9.com/Storage/8a9cd770-d1eb-40fc-b263-5da210bdb2bc",
    "priceType": "Fixed",
    "price": "654",
    "title": "Server One (4TB)",
    "disabled": false,
    "index": "4"
  },
  {
    "description": "Raspberry Pi (8GB RAM) single board computer with 128GB high endurance microSD flash storage in custom 3D printed case from CryptoCloaks with active fan cooling.",
    "id": "server-lite-128-cc",
    "image": "https://btcpay.start9.com/Storage/7a45d1e7-e1b7-4c3b-ad11-4cdea654a972",
    "priceType": "Fixed",
    "price": "227",
    "title": "Server Lite - Limited Edition Crypto Cloaks (128GB)",
    "disabled": false,
    "index": "5"
  },
  {
    "description": "Raspberry Pi (8GB RAM) single board computer with 256GB high endurance microSD flash storage in custom 3D printed case from CryptoCloaks with active fan cooling.",
    "id": "server-lite-256-cc",
    "image": "https://btcpay.start9.com/Storage/7a45d1e7-e1b7-4c3b-ad11-4cdea654a972",
    "priceType": "Fixed",
    "price": "245",
    "title": "Server Lite - Limited Edition Crypto Cloaks (256GB)",
    "disabled": false,
    "index": "6"
  },
  {
    "description": "Raspberry Pi (8GB RAM) single board computer with 128GB high endurance microSD flash storage and passive cooling.",
    "id": "server-lite-128",
    "image": "https://btcpay.start9.com/Storage/ede95175-e6c4-4165-964b-e09fc6d27c3c",
    "priceType": "Fixed",
    "price": "199",
    "title": "Server Lite (128GB)",
    "disabled": false,
    "index": "7"
  },
  {
    "description": "Raspberry Pi (8GB RAM) single board computer with 256GB high endurance microSD flash storage and passive cooling.",
    "id": "server-lite-256",
    "image": "https://btcpay.start9.com/Storage/ede95175-e6c4-4165-964b-e09fc6d27c3c",
    "priceType": "Fixed",
    "price": "217",
    "title": "Server Lite (256GB)",
    "disabled": false,
    "index": "8"
  },
  {
    "id": "cadet cap",
    "image": "https://btcpay.start9.com/Storage/6af29fb2-34fa-417e-8958-3599438caf07",
    "priceType": "Fixed",
    "price": "15",
    "title": "Cadet Cap",
    "disabled": false
  },
  {
    "id": "flatbill hat",
    "image": "https://btcpay.start9.com/Storage/5d4e04b4-d798-4a97-b1c7-d40f634d301f",
    "priceType": "Fixed",
    "price": "22",
    "title": "Flatbill hat",
    "disabled": false
  },
  {
    "id": "baseball hat",
    "image": "https://btcpay.start9.com/Storage/c71d220e-7f97-4670-888b-a01e8fecba64",
    "priceType": "Fixed",
    "price": "15",
    "title": "Baseball Hat",
    "disabled": false
  },
  {
    "description": "Enter sizes at checkout",
    "id": "men's \"sovereign computing\" t-shirt (crew neck)",
    "image": "https://btcpay.start9.com/Storage/2d2a2d98-ee5a-482e-8a77-d5461ea70f86",
    "priceType": "Fixed",
    "price": "20",
    "title": "Men's \"Sovereign Computing\" Crew",
    "disabled": false,
    "index": "12"
  },
  {
    "description": "Enter sizes at checkout",
    "id": "men's \"sovereign computing\" t-shirt (v-neck)",
    "image": "https://btcpay.start9.com/Storage/2fc4aa45-df98-4129-b50e-c3e4d73c0451",
    "priceType": "Fixed",
    "price": "20",
    "title": "Men's \"Sovereign Computing\" V-Neck",
    "disabled": false,
    "index": "13"
  },
  {
    "description": "Enter sizes at checkout",
    "id": "men's \"order from chaos\" t-shirt (crew neck)",
    "image": "https://btcpay.start9.com/Storage/f13b6a19-ced5-4e39-9c96-8a4ffb2e202c",
    "priceType": "Fixed",
    "price": "20",
    "title": "Men's \"Order From Chaos\" Crew",
    "disabled": false,
    "index": "14"
  },
  {
    "description": "Enter sizes at checkout",
    "id": "women's \"sovereign computing\" t-shirt",
    "image": "https://btcpay.start9.com/Storage/ee163fbc-cebd-4edc-a5ac-357bc0828e57",
    "priceType": "Fixed",
    "price": "20",
    "title": "Women's \"Sovereign Computing\" Crew",
    "disabled": false,
    "index": "15"
  },
  {
    "description": "Enter sizes at checkout",
    "id": "women's \"order from chaos\" t-shirt",
    "image": "https://btcpay.start9.com/Storage/7731f7c4-d3ac-4528-9ab9-bf900b30efe2",
    "priceType": "Fixed",
    "price": "20",
    "title": "Women's \"Order From Chaos\" Crew",
    "disabled": false,
    "index": "16"
  }
]
```
</details>

**Comment by elvece** [comment]
Fixed in 1.11.1. Thanks all for the quick response! All good on our end, cheers.
